src/ResourceManager/Network/Commands.Network/ExpressRouteCircuit/Stats/GetAzureExpressRouteCircuitARPTableCommand.cs

```diff
@@ -33,15 +33,7 @@ public enum DevicePathEnum
  		
      [Cmdlet(VerbsCommon.Get, "AzureRmExpressRouteCircuitARPTable"),OutputType(typeof(PSExpressRouteCircuitArpTable))]
      public class GetAzureExpressRouteCircuitARPTableCommand : NetworkBaseCmdlet		
-     {		
-         [Alias("ResourceName")]		
```

Comment:
You need to move this alias, and add an alias for Name to 'ExpressRouteCircuitName'.  This applies to all of your cmdlet changes.